bcda/client/bluebutton.go

```diff
@@ -256,3 +259,13 @@ func HashIdentifier(toHash string) (hashedValue string) {
 	}
 	return hex.EncodeToString(pbkdf2.Key([]byte(toHash), pepper, blueButtonIter, 32, sha256.New))
 }
+
+func UpdateParamWithLastUpdated(params *url.Values, since string, jobCreation time.Time) {
+	// only set the lower bound parameter if it exists and begins with "ge" (to align with what is expected in _lastUpdated)
+	if len(since) > 0 && strings.HasPrefix(since, "ge") {
+		params.Set("_lastUpdated", since)
```

Comment:
The lower bound should be a `gt`. Here's the relevant FHIR specification 

> Resources will be included in the response if their state has changed after the supplied time (e.g. if Resource.meta.lastUpdated is later than the supplied _since time).

Reply:
Yup!  Updated.